sssom/constants.py

```diff
@@ -177,3 +166,43 @@ class SchemaValidationType(str, Enum):
     SchemaValidationType.JsonSchema,
     SchemaValidationType.PrefixMapCompleteness,
 ]
+
+
+class SSSOMSchemaViewSingleton(object):
+    """Singleton class that holds the SSSOM schema view."""
+
+    def __new__(cls):
+        """Create a instance of the SSSOM schema view if non-existent."""
+        if not hasattr(cls, "instance"):
+            cls.instance = super(SSSOMSchemaViewSingleton, cls).__new__(cls)
+            return cls.instance
+
+
+class SSSOMSchemaView(SSSOMSchemaViewSingleton):
+    """
+    SchemaView class from linkml which is instantiated when necessary.
+
+    Reason for this: https://github.com/mapping-commons/sssom-py/issues/322
+    Implemented via PR: https://github.com/mapping-commons/sssom-py/pull/323
+    """
+
+    def __init__(self):
+        """Initialize class attributes."""
+        self.view = SchemaView(SCHEMA_YAML)
+        self.dict = schema_as_dict(self.view.schema)
```

Comment:
so it looks like you backed out the changes that used lazy loading of these

the whole point of having an abstraction over SchemaView was to allow deferred loading...

Reply:
I was testing it with minimal code and it seemed to run tests fractionally faster. I've rolled it back to the lazy loading.